docling_core/types/doc/document.py

```diff
@@ -1309,6 +1310,17 @@ def export_to_dict(self) -> Dict:
         """export_to_dict."""
         return self.model_dump(mode="json", by_alias=True, exclude_none=True)
 
+    def save_to_json_file(self, document_json_path, indent: int=4) -> int:
+        """export_to_json."""
+        with open(document_json_path, 'w') as f:
+            return f.write(self.model_dump_json(indent=indent))
+    
+    @classmethod
+    def load_from_json_file(cls, document_json_path):
+        """load_from_json."""
+        with open(document_json_path, 'r') as f:
+            return cls(**json.load(f))
```

Comment:
Please use the official pydantic way to load a model from JSON, which would be:
```
cls.model_validate_json(f.read())
```

see https://docs.pydantic.dev/latest/api/base_model/#pydantic.BaseModel.model_validate_json

Reply:
Done, thank you!